common/constants.py

```diff
@@ -0,0 +1,11 @@
+"""
```

Comment:
This file will need some second/third opinions. Basically we need these values in our URL definitions, and as Django settings they caused errors when used in `AppConfig` definitions. These values make more sense as constants than settings anyway. Up for debate: the necessity of this file, the location of this file, and when the corresponding values should be removed from the settings files (they were not removed as part of this PR to keep the diff size reasonable)

Reply:
@gsidebo I believe @bderusha is moving these constants out of settings in his discussions PR: https://github.com/edx/edx-platform/pull/17383